app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -227,6 +227,10 @@ input[type="checkbox"]{
   float: right;
 }
 
+.datetime.month, .datetime.day, .datetime.year{
```

Comment:
This is really minor, but maybe we could refactor the code like:

```scss
.datetime{
  &.month, &.day, &.year{
    margin-bottom: $base-margin;
  }
}
```

I think this is functionally equivalent.